docs/fieldnorm/verification_validation.md

```diff
@@ -1,7 +1,70 @@
 
-Donec mi mi, suscipit venenatis rutrum eu, gravida consectetur justo. Nunc neque ipsum, tincidunt ac libero eu, tincidunt accumsan lorem. Ut aliquet maximus nibh, id volutpat nisi efficitur nec. Nulla et feugiat ipsum. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Curabitur egestas est ac augue hendrerit finibus. Pellentesque hendrerit, magna gravida tincidunt bibendum, velit leo finibus nisi, non sagittis dui purus vitae orci. Etiam lobortis diam sed sem congue lacinia. Phasellus in fermentum libero.
+The following citation is taken from IMDRF/SaMD WG/N23 FINAL:2015 "Software as a Medical Device (SaMD): Application of Quality Management System "
 
-Donec id nibh aliquam, euismod velit id, aliquam ante. Vestibulum finibus interdum tellus quis efficitur. Quisque convallis nec nisl ut sollicitudin. Sed fringilla tellus et massa viverra interdum. Curabitur egestas diam at dui consectetur, et dapibus dui dignissim. Maecenas eget pulvinar enim. Etiam sodales id dui id venenatis. Duis id pellentesque tellus. Quisque enim justo, tristique id volutpat in, euismod ac tellus. Etiam faucibus velit sed purus commodo, eu vehicula velit egestas. Vivamus faucibus est et tortor imperdiet convallis. Vestibulum nulla enim, blandit a purus eget, cursus pellentesque lectus. Nulla finibus mi euismod eros lobortis, vitae eleifend erat rutrum.
\ No newline at end of file
+> Verification and Validation
+> The verification and validation (V&V) activities should be targeted towards the criticality and impact to patient safety of the SaMD, as discussed in IMDRF/SaMD WG/N12.Typically, verification (providing assurance that the design and development activity at each development stage conforms to the requirements) and validation (providing reasonable confidence that the software meets its intended use/user needs and operational requirements) activities ensure that all elements from the SaMD design and development—including any changes made during maintenance/upgrades—have been implemented correctly and that objective evidence of this implementation is recorded.
+
+In short, Barry Boehm expressed the difference between the software verification and software validation as follows:
+
+> Verification: Are we building the product right?
+> Validation: Are we building the right product?
+
+Often these definitions do not match with the definitions used in the medical laboratory. However, in terms of software development and thus bioinformatics, following software verification and software validation definitions make more sense. Noteworthy, the ISO-15189 states that validation is needed for laboratory designed or developed methods and for validated methods subsequently modified. Both are true for software development.
+
+The IEC-62304 mentiones different aspects of verifation; i.e. software unit verification and software integration testing, and validation; i.e. software system test.
```

Comment:
typo: verifation -> verification

Reply:
This gives a lot of background of what other people think about validation/verificaiton, but what I miss is a clear statement of wat *we* expect.

Reply:
I would love to say "this is what you need to do!" can we agree on that?

Reply:
added: Verification and Validation of software should be performed as described below.